ReleaseChecklist.md

```diff
@@ -6,6 +6,7 @@
  - [ ] DockerHub account with push rights to the [``solc`` image](https://hub.docker.com/r/ethereum/solc).
  - [ ] Launchpad (Ubuntu One) account with a membership in the ["Ethereum" team](https://launchpad.net/~ethereum) and
        a gnupg key for your email in the ``ethereum.org`` domain (has to be version 1, gpg2 won't work).
+ - [ ] Dependencies of the Docker script: ``docker-buildx``
```

Comment:
These could have different names in different distros so I'd specify which one we mean:

```suggestion
 - [ ] Ubuntu/Debian dependencies of the Docker script: ``docker-buildx``.
```

I see that on Arch it's called that too, but have not checked any others so I'd not necessarily claim it's always the name :)

Reply:
Fair enough, limiting ourselves to ubuntu/debian is at least consistent and definitely correct:)